analytic_engine/src/sst/parquet/writer.rs

```diff
@@ -97,7 +97,7 @@ struct RecordBatchGroupWriter {
     input_exhausted: bool,
     // Time range of rows, not aligned to segment.
     real_time_range: Option<TimeRange>,
-    column_values: Vec<Option<ColumnValueSet>>,
+    column_values: Option<Vec<Option<ColumnValueSet>>>,
```

Comment:
Add some comments to explain this field.

Reply:
Fixed